### DISCUSSION TITLE
动态-直播信息扩充  功能建议

### DISCUSSION THREAD

**Discussion by fengsha990817** [the1812/Bilibili-Evolved#1361] [功能建议 / Ideas]
希望能够让”正在直播“这一栏填满上下的所有空间，并且能够根据页面的缩放自适应长度。
![Snipaste_2020-12-23_21-11-39](https://user-images.githubusercontent.com/38030455/102999210-97746a80-4563-11eb-92c0-ef8d58d8485c.png)

**Comment by the1812** [comment]
`直播信息扩充`就有这功能
